tensorboardX/embedding.py

```diff
@@ -23,20 +23,21 @@ def make_tsv(metadata, save_path, metadata_header=None):
 # https://github.com/tensorflow/tensorboard/issues/44 image label will be squared
 def make_sprite(label_img, save_path):
     import math
-    import torch
-    import torchvision
+    import numpy as np
     from .x2num import make_np
+    from .utils import make_grid
+    from PIL import Image
     # this ensures the sprite image has correct dimension as described in
     # https://www.tensorflow.org/get_started/embedding_viz
     nrow = int(math.ceil((label_img.size(0)) ** 0.5))
```

Comment:
.size(0) does not work on numpy arrays https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.size.html